src/components/CodeInput.tsx

```diff
@@ -1,272 +1,122 @@
-import React, { useLayoutEffect } from 'react'
-import {
-  ActivityIndicator,
-  LayoutAnimation,
-  Platform,
-  StyleProp,
-  StyleSheet,
-  Text,
-  View,
-  ViewStyle,
-} from 'react-native'
+import React from 'react'
+import { ActivityIndicator, StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native'
 import Card from 'src/components/Card'
-import ClipboardAwarePasteButton from 'src/components/ClipboardAwarePasteButton'
-import TextInput, { LINE_HEIGHT } from 'src/components/TextInput'
+import TextInput from 'src/components/TextInput'
 import Checkmark from 'src/icons/Checkmark'
-import InfoIcon from 'src/icons/InfoIcon'
+import ExclamationCircleIcon from 'src/icons/ExclamationCircleIcon'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
-import { Shadow, Spacing } from 'src/styles/styles'
-import { useClipboard } from 'src/utils/useClipboard'
+import { Spacing } from 'src/styles/styles'
 
 export enum CodeInputStatus {
-  Disabled = 'Disabled', // input disabled
   Inputting = 'Inputting', // input enabled
-  Received = 'Received', // code received, still not validated
   Processing = 'Processing', // code validated, now trying to send it
   Error = 'Error', // the code processing failed and it's waiting to be input again.
   Accepted = 'Accepted', // the code has been accepted and completed
 }
 
 export interface Props {
-  label?: string | null
   status: CodeInputStatus
   inputValue: string
   inputPlaceholder: string
-  inputPlaceholderWithClipboardContent?: string
   onInputChange: (value: string) => void
-  shouldShowClipboard: (value: string) => boolean
-  multiline?: boolean
-  numberOfLines?: number
   testID?: string
   style?: StyleProp<ViewStyle>
-  shortVerificationCodesEnabled?: boolean
   autoFocus?: boolean
 }
```

Comment:
Many of these props were only used for import wallet, which is why I opted to refactor into a new component